providers/aws-ec2/src/main/java/org/jclouds/aws/ec2/options/AWSRunInstancesOptions.java

```diff
@@ -61,6 +61,22 @@ public AWSRunInstancesOptions inPlacementGroup(String placementGroup) {
    }
 
    /**
+    * Specifies the tenancy of the instance within which to launch the instance(s).
+    */
+   public AWSRunInstancesOptions withTenancy(String tenancy) {
```

Comment:
Looking at the docs, it looks like the only allowed values are `default | dedicated | host`. I think it is worth creating an enum so users don't have the chance to pass in the wrong values (the same enum should be used here and in the aws template options.